lib/std/os/test.zig

```diff
@@ -880,53 +880,72 @@ test "POSIX file locking with fcntl" {
         return error.SkipZigTest;
     }
 
-    if (true) {
-        // https://github.com/ziglang/zig/issues/11074
-        return error.SkipZigTest;
-    }
-
     var tmp = std.testing.tmpDir(.{});
     defer tmp.cleanup();
 
     // Create a temporary lock file
     var file = try tmp.dir.createFile("lock", .{ .read = true });
     defer file.close();
-    try file.setEndPos(2);
+    try file.setEndPos(4);
     const fd = file.handle;
 
-    // Place an exclusive lock on the first byte, and a shared lock on the second byte:
-    var struct_flock = std.mem.zeroInit(std.os.Flock, .{ .type = std.os.F.WRLCK });
-    _ = try std.os.fcntl(fd, std.os.F.SETLK, @ptrToInt(&struct_flock));
-    struct_flock.start = 1;
-    struct_flock.type = std.os.F.RDLCK;
+    // The first byte (0) will be used to test an exclusive lock.
+    // The second byte (1) will be used to test a shared lock.
+    // The third byte (2) will be locked by the child to signal the parent
+    // to do the test.
```

Comment:
Either s/the test/its tests or s/the test/the tests.

Reply:
I've made that change locally, but I'll hold off commit to avoid triggering the checks again with just comments.